.github/workflows/test-dream-conda.yml

```diff
@@ -1,15 +1,23 @@
 name: Test Dream with Conda
 on:
-  workflow_dispatch
+  pull_request_target:
+    types:
+      - closed
+    branches:
+      - 'main'
```

Comment:
I just realized that this should include 'development' 😫... We (hopefully won't need to) push hotfix PRs to 'main', so we still need testing there)

Context switching between "discussing the ideal" and "real-world implementation" 🤷